project-base/tests/ShopBundle/Acceptance/acceptance/CartCest.php

```diff
@@ -105,14 +115,19 @@ public function testAddToCartFromProductDetail(
         FloatingWindowPage $floatingWindowPage
     ) {
         $me->wantTo('add product to cart from product detail');
-        $me->amOnPage('/22-sencor-sle-22f46dm4-hello-kitty/');
-        $me->see('Add to cart');
+        // 22-sencor-sle-22f46dm4-hello-kitty
+        $me->amOnLocalizedRoute('front_product_detail', ['id' => 1]);
+        $me->seeTranslationFrontend('Add to cart');
         $productDetailPage->addProductIntoCart(3);
-        $me->see('Product 22" Sencor SLE 22F46DM4 HELLO KITTY (3 pcs) added to the cart');
+        $me->seeTranslationFrontend('Product <strong>%name%</strong> (%quantity% %unitName%) added to the cart', 'messages', [
+            '%name%' => t('22" Sencor SLE 22F46DM4 HELLO KITTY', [], 'dataFixtures', $me->getFrontendLocale()),
+            '%quantity%' => 3,
+            '%unitName%' => $me->getDefaultUnitName(),
+        ]);
         $floatingWindowPage->closeFloatingWindow();
-        $cartBoxPage->seeInCartBox('1 item for CZK10,497.00');
-        $me->amOnPage('/cart/');
-        $me->see('22" Sencor SLE 22F46DM4 HELLO KITTY');
+        $cartBoxPage->seeInCartBox(1, '10497');
```

Comment:
Do we have some other tests, that test price formatting by domain?

Reply:
We have `NumberFormatterTest` - it tests formatting for english locale